packages/staking/src/features/Drawer/confirmation/StakePoolConfirmationContent.tsx

```diff
@@ -14,11 +17,18 @@ import { StakePoolConfirmationBody } from './StakePoolConfirmationBody';
 import styles from './StakePoolConfirmationContent.module.scss';
 import { StakePoolDepositReclaimDetails } from './StakePoolDepositReclaimDetails';
 
+const TX_VALIDITY_PERIOD = process.env.SHARED_WALLET_TX_VALIDITY_INTERVAL;
```

Comment:
IMO we should not introduce many names for the same thing as it creates a confusion.

```suggestion
const SHARED_WALLET_TX_VALIDITY_INTERVAL = process.env.SHARED_WALLET_TX_VALIDITY_INTERVAL;
```

Reply:
done here 4f7a023ca8a159613b8b0dab0dec664d4d2a2454